config/deploy.rb

```diff
@@ -37,4 +39,6 @@
     end
   end
 end
+
+before 'deploy:reverted', 'npm:install'
```

Comment:
Added because of https://github.com/capistrano/npm/issues/7#issuecomment-274749621